cmd/snap/cmd_known.go

```diff
@@ -34,6 +36,8 @@ type cmdKnown struct {
 		AssertTypeName string   `required:"true"`
 		HeaderFilters  []string `required:"0"`
 	} `positional-args:"true" required:"true"`
+
+	Store bool `long:"store"`
```

Comment:
@chipaca has a good point about store being a verb too, giving a plausible but wrong meaning to the flag.

Let's go with `--remote` as you originally suggested.